backend/test/logger.test.js

```diff
@@ -66,26 +67,45 @@ describe('logger', () => {
         const arg2 = Error();
         arg2.stack = 'secondArgStack';
         logger.info(arg1, arg2);
-        // eslint-disable-next-line no-console
-        expect(console.log).toHaveBeenCalledWith(expect.stringMatching(`${arg1} ${arg2.stack}`));
+        expect(console.log).toHaveBeenCalledWith(expect.stringContaining(`${arg1} ${arg2.stack}`));
     });
 
     it('should handle plain objects and array arguments', () => {
         const logger = initLogging(commonConfig).getLogger('objects');
         global.console = { log: jest.fn() };
 
-        /* eslint-disable no-console */
         logger.info({});
-        expect(console.log).toHaveBeenLastCalledWith(expect.stringMatching('{}'));
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('{}'));
 
         logger.info({ a: 'b' });
-        expect(console.log).toHaveBeenLastCalledWith(expect.stringMatching('{"a":"b"}'));
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('{"a":"b"}'));
 
         logger.info([1, 'a', { b: 'c' }, true]);
-        expect(console.log).toHaveBeenLastCalledWith(expect.stringMatching('[1,"a",{"a":"b"},true]'));
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('[1,"a",{"b":"c"},true]'));
 
         logger.info(new Set());
-        expect(console.log).toHaveBeenLastCalledWith(expect.stringMatching('[object Set]'));
-        /* eslint-enable no-console */
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('{}'));
+    });
+
+    it('should handle primitive arguments', () => {
+        const logger = initLogging(commonConfig).getLogger('parsability');
+        global.console = { log: jest.fn() };
+
+        logger.info(1.45, true, 'test', null, undefined);
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('1.45 true test null '));
+    });
+
+    it('should handle arguments not parsable to string', () => {
+        const logger = initLogging(commonConfig).getLogger('parsability');
+        global.console = { log: jest.fn() };
+
+        const circularObject = {};
+        circularObject.ref = circularObject;
+        logger.info(circularObject);
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('<NotParsable>'));
+
+        const bigInt = BigInt(123);
+        logger.info(bigInt);
+        expect(console.log).toHaveBeenLastCalledWith(expect.stringContaining('<NotParsable>'));
```

Comment:
These are the only cases in which JSON.stringify will throw an Error according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#exceptions